### PR TITLE
cleaning

### PR DESCRIPTION
**Pull request by EmmaRenauld**

### CODE REVIEW

File: dwi_ml/tracking/step_tracker.py [hunk 1]
```diff
@@ -1,21 +1,19 @@
...
 import nibabel as nib
+from nibabel.streamlines import Tractogram
 import numpy as np
 import torch
-from nibabel.streamlines import Tractogram
 
 from dwi_ml.data.dataset.single_subject_containers import (
     MRIDataVolume)
-from dwi_ml.data.processing.dwi.interpolation import (
+from dwi_ml.data.processing.volume.interpolation import (
```
Comment (AntoineTheb) [inline dwi_ml/tracking/step_tracker.py]: super minor but the imports should be ordered alphabetically

File: dwi_ml/tracking/tracker_abstract.py [hunk 2]
```diff
@@ -19,8 +18,7 @@
     MRIDataVolume, SubjectData)
 from dwi_ml.tracking.step_tracker import (
     StepTracker, PreInitializedStepTracker)
-from dwi_ml.data.processing.space.utils import (
-    convert_mm2vox)
+from dwi_ml.data.processing.space.world_to_vox import convert_world_to_vox
```
Comment (AntoineTheb) [inline dwi_ml/tracking/tracker_abstract.py]: again super minor but the imports could be re-ordered

File: dwi_ml/io/TO_SEND_TO_SCILPY.py [hunk 3]
```diff
@@ -1,97 +0,0 @@
...
-from typing import Tuple
-
-import nibabel as nib
-import numpy as np
-from dipy.core.gradients import GradientTable, gradient_table as load_gradient_table
-
-from scil_vital.shared.code.transformation.volume import resample_volume
-
-
-def load_dwi(dwi_file: str) -> nib.Nifti1Image:
```
Comment (AntoineTheb) [inline dwi_ml/io/TO_SEND_TO_SCILPY.py]: I couldn't find the functions in this file; I'm sure they exist somewhere in scilpy, what would their names be ?

File: dwi_ml/tracking/tracker_abstract.py (hunk 2 above)
Comment (AntoineTheb) [reply dwi_ml/tracking/tracker_abstract.py]: I've only commented of these two files about import ordering but please verify the other files also

File: dwi_ml/tracking/step_tracker.py [hunk 4]
```diff
@@ -1,21 +1,19 @@
...
     torch_trilinear_interpolation)
-from dwi_ml.data.processing.space.utils import (
-    convert_mm2vox)
-from dwi_ml.data.processing.dwi.neighbourhood import  (
-    get_interp_neighborhood_vectors)
+from dwi_ml.data.processing.space.world_to_vox import convert_world_to_vox
+from dwi_ml.data.processing.space.neighbourhood import  (
+    get_neighborhood_vectors_axes
```
Comment (AntoineTheb) [inline dwi_ml/tracking/step_tracker.py]: Missing parenthesis

File: dwi_ml/io/TO_SEND_TO_SCILPY.py (hunk 3 above)
Comment (EmmaRenauld) [reply dwi_ml/io/TO_SEND_TO_SCILPY.py]: Thanks for pointing this out. We haven't really dealt with this function, it's true.

Hum... Actually, the only real line here is nib.load. The rest is a check of the name of the file. It shouldn't really be an "official" function, just as part of a script, I think.

I will bring back this file but rename it to "to_clean".

**Comment by EmmaRenauld** [comment]
I won't arrange all Pep8 (unused imports) because they are in files that have not been verified yet. I changed the imports to keep track of were the functions now are but we will have to re-code these functions anyway. Merging.

**Pull request merged**